.circleci/config.yml

```diff
@@ -103,25 +103,18 @@ jobs:
       - run:
           name: Commit and push
           command: |
-            dnf install -y ca-certificates git git-lfs openssh-clients python3-rpm python3-debian rpmdevtools
+            dnf install -y ca-certificates git git-lfs openssh-clients
             git clone git@github.com:freedomofpress/securedrop-yum-test.git
             cd securedrop-yum-test
             git lfs install
             git config user.email "securedrop@freedom.press"
             git config user.name "sdcibot"
             mkdir -p workstation/dom0/f32-nightlies
-            # Copy the new packages over and cleanup the old ones
+            # Copy the new packages over
             cp -v /tmp/workspace/*.rpm workstation/dom0/f32-nightlies/
-            ~/project/scripts/clean-old-packages workstation/dom0/f32-nightlies 7
             git add .
             # If there are changes, diff-index will fail, so we commit
             git diff-index --quiet HEAD || git commit -m "Automated SecureDrop workstation build"
-            # And clean up non-nightly packages too
-            ~/project/scripts/clean-old-packages workstation/dom0/f32 4
-            git add .
-            git diff-index --quiet HEAD || git commit -m "Cleanup old packages"
-
-            git push origin main
```

Comment:
This push is still needed, because it's what pushes the nightly commit.

Reply:
Oof, of course, sorry about that & thanks for the catch.